### DISCUSSION TITLE
Premade PvP/Joining as groups

### DISCUSSION THREAD

**Discussion by ghost** [chromiecraft/chromiecraft#3550] [Suggestions]
Hello everyone, my nickname on the server Mircalla.

I'm making this post because I believe there will be more people feeling the same frustration as I have. What I'm talking about joining pvp as premades. First of all I completely understand people who wants to play together with their friends and I know that the population who play PvP in this server is pretty small.

We all know how unbalanced the pvp is currently due to Wotlk talents and skill trees and there's no need to argue that and premade pvp (specially in WSG) making this much worse because sometimes in a game only one side having a premade team and the other side is completly randomized. This frustration will keep people away from pvp even more so now than before, including me who is almost active all day.

But I have to say that I'm still not entirely against the premade Idea. I think people should be able to play with their friends together naturally, therefore I wanted to make a suggestion, so at least we can talk about this frustration and find better solutions to the issue.

My suggestion is to limit groups and it should differ from BG to BG if its possible to do so of course because I dont understand about coding/programming. So limit how? For example 2 player(duo) max or no premade at all for WSG and 3 or higher for Arathi and AV. If you have better Idea or against this or if you support this, please share your opinion and ideas. I would really like to hear and make devs hear about this, so that we can play and enjoy our times better when we do.

**Comment by 55Honey** [comment]
> My suggestion is to limit groups...

Hello @Arisatheus,
thanks for your suggestion.
Please note that there is a global limit of 3 players per premade party. A party of 4 or more can not queue together.
